cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -169,26 +182,32 @@ MapBuilderBridge::GetTrajectoryStates() {
     const int trajectory_id = entry.first;
     const SensorBridge& sensor_bridge = *entry.second;
 
-    const cartographer::mapping::TrajectoryBuilder* const trajectory_builder =
-        map_builder_.GetTrajectoryBuilder(trajectory_id);
-    if (trajectory_builder == nullptr) {
-      continue;
+    std::unique_ptr<TrajectoryState::LocalSlamData>
+        current_trajectory_state_data;
+    {
+      cartographer::common::MutexLocker lock(&mutex_);
+      if(trajectory_state_data_.count(trajectory_id) == 0) {
```

Comment:
Style: Space after the `if`. `clang-format`?

Reply:
Sure. But Wally now takes care of this, even if I miss it.